### ISSUE TITLE
The DPO 'grad_norm': 0.0,

### ISSUE THREAD

**Issue by Faded1022** [huggingface/trl#1778]
{'loss': 0.3537, 'grad_norm': 0.0, 'learning_rate': 5e-06, 'rewards/chosen': 1.109375, 'rewards/rejected': -0.08642578125, 'rewards/accuracies': 0.8500000238418579, 'rewards/margins': 1.1953125, 'logps/rejected': -20.25, 'logps/chosen': -46.5, 'logits/rejected': -2.796875, 'logits/chosen': -1.96875, 'epoch': 0.05}

**Comment by qgallouedec** [comment]
Thank you for reporting. However, I'm closing the issue because it doesn't provide enough information to reproduce what you're describing. If anyone encounters this situation, please open a new issue and provide the necessary information for it to reproduce.
